core/domain/topic_domain_test.py

```diff
@@ -857,86 +745,88 @@ def test_update_thumbnail_filename(self):
+        self
+    ) -> None:
+        with self.assertRaisesRegexp(  # type: ignore[no-untyped-call]
             Exception, 'The subtopic with id invalid_id does not exist.'):
-            self.topic.update_subtopic_title('invalid_id', 'new title')
+            self.topic.update_subtopic_title('invalid_id', 'new title') # type: ignore[arg-type]
 
-    def test_update_subtopic_title(self):
+    def test_update_subtopic_title(self) -> None:
         self.assertEqual(len(self.topic.subtopics), 1)
         self.assertEqual(self.topic.subtopics[0].title, 'Title')
 
         self.topic.update_subtopic_title(1, 'new title')
         self.assertEqual(self.topic.subtopics[0].title, 'new title')
 
-    def test_update_subtopic_thumbnail_filename(self):
+    def test_update_subtopic_thumbnail_filename(self) -> None:
         self.assertEqual(len(self.topic.subtopics), 1)
         self.assertEqual(
             self.topic.subtopics[0].thumbnail_filename, 'image.svg')
         self.assertEqual(
             self.topic.subtopics[0].thumbnail_size_in_bytes, 21131)
 
         # Test Exception when the thumbnail is not found in filesystem.
-        with self.assertRaisesRegexp(
+        with self.assertRaisesRegexp(  # type: ignore[no-untyped-call]
             Exception, 'The thumbnail %s for subtopic with topic_id %s does'
             ' not exist in the filesystem.' % (
                 'new_image.svg', self.topic_id)):
             self.topic.update_subtopic_thumbnail_filename(1, 'new_image.svg')
 
         # Test successful update of thumbnail_filename when the thumbnail
         # is found in the filesystem.
-        with python_utils.open_file(
+        with python_utils.open_file(  # type: ignore[no-untyped-call]
             os.path.join(feconf.TESTS_DATA_DIR, 'test_svg.svg'), 'rb',
             encoding=None) as f:
             raw_image = f.read()
-        fs = fs_domain.AbstractFileSystem(
-            fs_domain.GcsFileSystem(
+        fs = fs_domain.AbstractFileSystem(  # type: ignore[no-untyped-call]
+            fs_domain.GcsFileSystem(  # type: ignore[no-untyped-call]
                 feconf.ENTITY_TYPE_TOPIC, self.topic_id))
-        fs.commit(
+        fs.commit(  # type: ignore[no-untyped-call]
             'thumbnail/new_image.svg', raw_image, mimetype='image/svg+xml')
         self.topic.update_subtopic_thumbnail_filename(1, 'new_image.svg')
         self.assertEqual(
             self.topic.subtopics[0].thumbnail_filename, 'new_image.svg')
         self.assertEqual(
             self.topic.subtopics[0].thumbnail_size_in_bytes, len(raw_image))
 
-        with self.assertRaisesRegexp(
+        with self.assertRaisesRegexp(  # type: ignore[no-untyped-call]
             Exception, 'The subtopic with id invalid_id does not exist.'):
             self.topic.update_subtopic_thumbnail_filename(
-                'invalid_id', 'new title')
+                'invalid_id', 'new title') # type: ignore[arg-type]
```

Comment:
Why is this ignore needed?

Reply:
Ditto elsewhere for the `# type: ignore[arg-type]`

Reply:
Previously, I added these to avoid such errors
![d_8](https://user-images.githubusercontent.com/89520981/148723678-5b8dd003-281b-48ec-aa0b-99edfafe6d5f.png)
but later I realized that we do not need these tests after adding type annotations so, I am removing them. Please correct me if I am wrong.

Reply:
Right.